$NetBSD: patch-po_Makefile.in.in,v 1.1 2013/08/21 14:08:08 richard Exp $

some misc fixes in order to get over the automake deprecated
`@mkdir_p@` as well as adverse effects of SHELL in this file.

--- po/Makefile.in.in.orig	2013-03-23 14:51:10.000000000 +0000
+++ po/Makefile.in.in
@@ -15,7 +15,6 @@ PACKAGE = @PACKAGE@
 VERSION = @VERSION@
 PACKAGE_BUGREPORT = @PACKAGE_BUGREPORT@
 
-SHELL = /bin/sh
 @SET_MAKE@
 
 srcdir = @srcdir@
@@ -39,10 +38,10 @@ INSTALL_DATA = @INSTALL_DATA@
 # In automake >= 1.10, @mkdir_p@ is derived from ${MKDIR_P}, which is defined
 # either as "/path/to/mkdir -p" or ".../install-sh -c -d". For these automake
 # versions, $(mkinstalldirs) and $(install_sh) are unused.
-mkinstalldirs = $(SHELL) @install_sh@ -d
-install_sh = $(SHELL) @install_sh@
+mkinstalldirs = @install_sh@ -d
+install_sh = @install_sh@
 MKDIR_P = @MKDIR_P@
-mkdir_p = @mkdir_p@
+mkdir_p = $(MKDIR_P)
 
 GMSGFMT_ = @GMSGFMT@
 GMSGFMT_no = @GMSGFMT@
